Description: Remove Windows-specific dependency
Author: Antonin Delpeuch <antonin@delpeuch.eu>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -44,14 +44,6 @@ version = "1"
 [target."cfg(unix)".dependencies.libc]
 version = "0.2"
 
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.59.0"
-features = [
-    "Win32_System_Console",
-    "Win32_Storage_FileSystem",
-    "Win32_Foundation",
-]
-
 [lints.clippy]
 dbg_macro = "warn"
 exhaustive_enums = "warn"
